app/views/components/markdown_editor/_blockquote.svg

```diff
@@ -1,3 +1,3 @@
-<svg class="app-c-markdown-editor__toolbar-icon" version="1.1" viewBox="0 0 50 50" width="50" height="50" aria-hidden="true">
+<svg version="1.1" viewBox="0 0 50 50" width="50" height="50">
```

Comment:
We should move these SVG files from views to the app/assets directory, since they aren't actually views and are image files.

So this file's new location should be `app/assets/images/components/markdown_editor/blockquote.svg` (note the removal of the underscore prefix) with similar for these other markdown components.